client/src/components/StreamField/blocks/ListBlock.test.js

```diff
@@ -113,7 +113,7 @@ describe('telepath: wagtail.blocks.ListBlock', () => {
         icon: 'placeholder',
         classname: null,
         helpText: 'use <strong>a few</strong> of these',
-        helpIcon: '<div class="icon-help">?</div>',
+        helpIcon: '<svg></svg>',
```

Comment:
Outside of tests, `helpIcon` renders an SVG icon since quite a few releases ago. I chose to use an empty `<svg>` element as the exact markup is irrelevant here.